### PR TITLE
[MTDSA-7855] - template for readme file

### PR DESCRIPTION
**Pull request by DreamcastKnight**

### CODE REVIEW

**Review by Mbrookesy — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,14 +1,42 @@
...
+**Add overview documentation here**
 
-TODO Describe microservice / API
+## Requirements 
+- Scala 2.12.x
+- Java 8
+- sbt 1.3.7
+- [Service Manager](https://github.com/hmrc/service-manager)
  
+## Development Setup
```
Comment (james-work-account) [inline README.md:11]:
```suggestion
## Development Se euoeutiuet
```

File: README.md [hunk 2]
```diff
@@ -1,14 +1,42 @@
...
+  
+Run from the console using: `sbt run` (starts on port XXXX by default)
+  
+Start the service manager profile: `sm --start MTDFB_XX`
+
 # Run tests
```
Comment (mashukreza) [inline README.md]:
```suggestion
# Running tests
```

File: README.md [hunk 3]
```diff
@@ -1,14 +1,42 @@
...
 ```
 sbt test
 sbt it:test
 ```
 
+## To view the RAML
```
Comment (mashukreza) [inline README.md]:
```suggestion
## Viewing RAML
```

File: README.md [hunk 4]
```diff
@@ -1,14 +1,42 @@
 API Example Microservice
 ========================
+**Add overview documentation here**
```
Comment (mashukreza) [inline README.md]:
```suggestion
**Add overview documentation here and use the title of the API and not the URL e.g. "Business details API" and not "business-details-api"**
```

**Review by mashukreza — COMMENTED** [review]
added suggested changes to template

**Pull request merged**